### ISSUE TITLE
rvc status results in a truncated log entry in rvd.log 

### ISSUE THREAD

**Issue by erikbor** [riboseinc/rvc#210]
Hi @jjr840430 

Running `rvc status` results in the following entry in `/var/log/rvd/rvd.log`:

```
2017-11-27T10:26:43HKT  INFO : CMD: Received command '{ "cmd": 3, "json": false, "param": "all" }'(at cmd.c:362)
2017-11-27T10:26:43HKT  INFO : CMD: Processing 'status' command(at cmd.c:281)
2017-11-27T10:26:43HKT  INFO : VPN: Getting connection status for VPN connection with name 'all'(at vpn.c:1301)
2017-11-27T10:26:43HKT  INFO : VPN: Getting status of VPN connection with name 'test1'(at vpn.c:1071)
2017-11-27T10:26:43HKT  INFO : VPN: Getting status of VPN connection with name 'test2'(at vpn.c:1071)
2017-11-27T10:26:43HKT  INFO : VPN: Getting status of VPN connection with name 'test3'(at vpn.c:1071)
2017-11-27T10:26:43HKT  INFO : VPN: Getting status of VPN connection with name 'test4'(at vpn.c:1071)
2017-11-27T10:26:43HKT  INFO : VPN: Getting status of VPN connection with name 'test5'(at vpn.c:1071)
2017-11-27T10:26:43HKT  INFO : VPN: Getting status of VPN connection with name 'test6'(at vpn.c:1071)
2017-11-27T10:26:43HKT  INFO : VPN: Getting status of VPN connection with name 'test7'(at vpn.c:1071)
2017-11-27T10:26:43HKT  INFO : VPN: Getting status of VPN connection with name 'test8'(at vpn.c:1071)
2017-11-27T10:26:43HKT  INFO : CMD: Sending response(at cmd.c:126)
2017-11-27T10:26:43HKT  INFO : CMD: Response string is 
'name: test1
        config-status: JSON configuration exists
        profile: /opt/rvc/etc/vpn.d/test1.ovpn
        status: DISCONNECTED
        openvpn-status: DISCONNECTED
        in-total: 0
        out-total: 0
        timestamp: 1511749603
        auto-connect: Disabled
        pre-exec-cmd: 
        pre-exec-status: 
        pre-exec-interval: 
name: test2
        config-status: No JSON configuration exists
        profile: /opt/rvc/etc/vpn.d/test2
        status: DISCONNECTED
        openvpn-status: DISCONNECTED
        in-total: 0
        out-total: 0
        timestamp: 1511749603
        auto-connect: Disabled
        pre-exec-cmd: 
        pre-exec-status: 
        pre-exec-interval: 
name: test3
        config-status: No JSON configuration exists
        profile: /opt/rvc/etc/vpn.d/test3
        status: DISCONNECTED
        openvpn-status: DISCONNECTED
        in-total: 0
        out-total: 0
        timestamp: 1511749603
        auto-connect: Disabled
        pre-exec-cmd: 
        pre-exec-status: 
        pre-exec-interval: 
name: test4
        config-status: JSON configuration exists
        profile: /opt/rvc/etc/vpn.d/test4.ovpn
        status: DISCONNECTED
        ope(at cmd.c:157)
2017-11-27T10:26:54HKT  INFO : CMD: Received command '{ "cmd": 3, "json": false, "param": "all" }'(at cmd.c:362)
2017-11-27T10:26:54HKT  INFO : CMD: Processing 'status' command(at cmd.c:281)
```

There are 8 VPNs configured but the log entry gets truncated at VPN4. Can you please please look into why the truncation takes place? Thanks

**Comment by jjr840430** [comment]
Fixed by #215

**Comment by ronaldtse** [comment]
Confirmed.
